### PR TITLE
Rename trailingComma to trailingCommaPHP

### PR DESCRIPTION
**Pull request by flexchar**
Rename `trailingComma` property to `trailingCommaPHP` in order to prevent collisions with other Prettier plugins/implementations.

As discussed in #964

### CODE REVIEW

File: src/printer.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const {
 function shouldPrintComma(options, level) {
   level = level || "none";
 
-  switch (options.trailingComma) {
+  switch (options.trailingCommaPHP) {
```
Comment (evilebottnawi) [inline src/printer.js]: Let's support also `options.trailingComma` for `all` and `none` for better integration

**Comment by flexchar** [comment]
Since 'none' is none despite option being present or not, I only add check for 'all'.

**Comment by evilebottnawi** [comment]
@czosel can you do release? because we are below 1.0 we can release this as minor, don't forget add infromation about this to release page

**Comment by czosel** [comment]
@evilebottnawi done. Is the text sufficient? https://github.com/prettier/plugin-php/releases/tag/v0.11.0

**Comment by evilebottnawi** [comment]
@czosel yes :+1:

**Comment by loilo** [comment]
What about the CLI option? Current v0.11.0 still uses `--trailing-comma` instead of `--trailing-comma-php`.

**Comment by evilebottnawi** [comment]
Need update too

**Comment by czosel** [comment]
Shall I try reverting the release or shall we publish a fix as 0.11.1?

**Comment by evilebottnawi** [comment]
@czosel yes, looks we do broken release, we should merge this and do release

**Comment by czosel** [comment]
:man_facepalming: I didn't realize that this wasn't merged yet.

**Comment by czosel** [comment]
I just unpublished 0.11.0 - the current master is good for release, right?

**Comment by evilebottnawi** [comment]
Yes

**Comment by czosel** [comment]
Published as 0.11.1

**Comment by loilo** [comment]
Alright, playground has been updated. 👍

**Comment by flexchar** [comment]
@evilebottnawi thanks for getting this out!
Did I miss CLI option as mentioned by @loilo? I can't seem to find any code for it thou...

**Comment by loilo** [comment]
I only tested with the 0.11.0 version @czosel published before merging the PR so I guess I *actually* tested 0.10.2 under the 0.11.0 label. 😁

Since there were no further commits for 0.11.1, I assume that your PR was completely fine @flexchar. ❤️
Sorry for the confusion.

**Comment by czosel** [comment]
I also apologize for the broken release - next time I won’t cut a release while sitting in a meeting 😅

**Pull request merged**